src/Tasks/Common/Resources/Strings.resx

```diff
@@ -879,4 +879,8 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1193: If PublishSelfContained is set, it must be either true or false. The value given was '{0}'.</value>
     <comment>{StrBegin="NETSDK1193: "}</comment>
   </data>
+    <data name="CannotHaveSolutionLevelOutputPath" xml:space="preserve">
+    <value>NETSDK1194: Building a solution with a specific OutputPath is not supported. Specifiy the output path at the individual project level instead.</value>
```

Comment:
It would be valuable to describe how to specify the output path at the project level. Would I use the `OutputPath` property in the project file for that?

Reply:
It felt too complicated to explain in the error message so I just simplified the text to:

> The "--output" option isn't supported when building a solution.